### PR TITLE
Replace function SVG with unicode character in Outline pane

### PR DESCRIPTION
**Pull request by iansu**
Associated Issue: #5262 

### Summary of Changes

* Replace function SVG icon with the unicode function symbol in the Outline pane

### Test Plan

- [x] Open the Outline pane and look at the function symbol to the left of any function name

### Screenshots

|Before|After|
|----------|------|
|Before|After|

### CODE REVIEW

File: src/components/PrimaryPanes/Outline.js [hunk 1]
```diff
@@ -55,8 +54,7 @@ export class Outline extends Component<Props> {
         className="outline-list__element"
         onClick={() => this.selectItem(location)}
       >
-        <Svg name="function" />
-        <PreviewFunction func={{ name, parameterNames }} />
+        λ <PreviewFunction func={{ name, parameterNames }} />
```
Comment (lukaszsobek) [inline src/components/PrimaryPanes/Outline.js]: We do potentially want a way to style this element individually, so a tag around it would be nice.

Comment (iansu) [reply src/components/PrimaryPanes/Outline.js]: Will do. I think that might also fix the failing test.

**Comment by jasonLaster** [comment]
should be an easy fix

**Comment by jasonLaster** [comment]
Welcome aboard @iansu!

**Comment by iansu** [comment]
I was just about to fix the failing test but it looks like you beat me to it. Thanks! 😀

**Comment by jasonLaster** [comment]
haha - no worries. mochitests are weird

**Pull request merged**